website/docs/r/batch_job_definition.html.markdown

```diff
@@ -256,6 +290,146 @@ The following arguments are optional:
 * `secret_name` - The name of the secret. The name must be allowed as a DNS subdomain name.
 * `optional` - (Optional) Specifies whether the secret or the secret's keys must be defined.
 
+### `ecs_properties`
+
+* `task_properties` - The properties of the Amazon ECS task definition on a job. See [`task_properties`](#task_properties) below.
+
+### `task_properties`
+
+* `containers` - The properties of the container. See [containers](#ecs_containers) below.
```

Comment:
This is actually a list of containers, right ?

Reply:
Yes, I fixed in https://github.com/hashicorp/terraform-provider-aws/pull/37871/commits/e90348c15176b284619438a3af227caec5ed34d4 and https://github.com/hashicorp/terraform-provider-aws/pull/37871/commits/10a4475ef5711adc94412978f3fc428e4f641758